projects/helper/unwrapLPs.js

```diff
@@ -499,10 +502,28 @@ async function unwrapUniswapLPs(balances, lpPositions, block, chain='ethereum',
       }
       await Promise.all(lpPositions.map(async lpPosition => {
         try{
+            let token0, token1, supply
             const lpToken = lpPosition.token
-            const token0 = (await tokens0).output.find(call=>call.input.target === lpToken).output.toLowerCase()
-            const token1 = (await tokens1).output.find(call=>call.input.target === lpToken).output.toLowerCase()
-            const supply = (await lpSupplies).output.find(call=>call.input.target === lpToken).output
+            const token0_ = (await tokens0).output.find(call=>call.input.target === lpToken)
+            const token1_ = (await tokens1).output.find(call=>call.input.target === lpToken)
+            const supply_ = (await lpSupplies).output.find(call=>call.input.target === lpToken)
+            const callFailed = [token0_, token1_, supply_].some(response => !response || !response.success)
+
+            if (skipFailingLPs && callFailed) {
```

Comment:
an issue here is that some calls can fail due to rpcs failing, and if this is true you will end up ignoring real tvl, it's much better to break the whole adapter to avoid ingesting wrong data in those cases

Reply:
I will revert this change and blacklist those LP tokens instead